DataFormats/Common/interface/MultiCollection.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DataFormats_Common_interface_MultiCollection_h
+#define DataFormats_Common_interface_MultiCollection_h
+
+#include <vector>
+
+#include "DataFormats/Common/interface/RefProd.h"
+
+namespace edm {
+  template <typename T>
+  using MultiCollection = std::vector<edm::RefProd<T>>;
```

Comment:
Repeating https://github.com/cms-sw/cmssw/pull/49190#issuecomment-3426519141 I still find the `MultiCollection` not to be a very descriptive name. How about e.g. `RefProdVector`?